### DISCUSSION TITLE
If a PDF has a virus, will viewing it in react-pdf activate it.

### DISCUSSION THREAD

**Discussion by graphei** [wojtekmaj/react-pdf#1745] [Q&A] [answered]
If a PDF has a virus, will viewing it in react-pdf activate it.

**Comment by wojtekmaj** [comment, marked as answer]
Possibly. I absolutely cannot guarantee it won't. However, PDF.js (Firefox's PDF rendering engine that we also use) architecture is vastly different from Adobe Reader's and I would expect the latter to be an attack vector far more commonly.
